docs/project/development.md

```diff
@@ -1,171 +1,125 @@
-$ export PATH="$PATH:$(brew --prefix llvm@15)/bin"
-$ export LDFLAGS="$LDFLAGS -L$(brew --prefix llvm@15)/lib"
-$ export CPPFLAGS="$CPPFLAGS -I$(brew --prefix llvm@15)/include"
-```
-
-### Install Zig
+In addition to this, you will need either `bun` or `npm` installed to install the package.json dependencies.
 
-{% callout %}
-**⚠️ Warning** — You must use the same version of Zig used by Bun in [oven-sh/zig](https://github.com/oven-sh/zig). Installing with `brew` or via Zig's download page will not work!
-{% /callout %}
+## Install Zig
 
-Use [`zigup`](https://github.com/marler8997/zigup) to install the version of Zig (`ZIG_VERSION`) specified in the official [`Dockerfile`](https://github.com/oven-sh/bun/blob/main/Dockerfile). For example:
+Zig can installed either with our npm package [`@oven/zig`](https://www.npmjs.com/package/@oven/zig), or by using [zigup](https://github.com/marler8997/zigup).
 
-```bash
-$ zigup 0.11.0-dev.1783+436e99d13
+```
+$ bun install -g @oven/zig
+$ zigup master
 ```
 
-### Building
+## Building
 
-To install and build dependencies:
+After cloning the repository, prepare bun to be built:
 
 ```bash
-# without --depth=1 this will take 20+ minutes on 1gbps internet
-# mostly due to WebKit
-$ git submodule update --init --recursive --progress --depth=1 --checkout
-$ bun install
-$ make vendor identifier-cache
+$ make setup
 ```
 
-To compile the C++ bindings:
+Then to build Bun:
 
 ```bash
-# without -j this will take 30+ minutes
-$ make bindings -j12
+$ make dev
 ```
 
-<!-- If you're building on a macOS device, you'll need to have a valid Developer Certificate, or else the code signing step will fail. To check if you have one, open the `Keychain Access` app, go to the `login` profile and search for `Apple Development`. You should have at least one certificate with a name like `Apple Development: user@example.com (WDYABC123)`. If you don't have one, follow [this guide](https://ioscodesigning.com/generating-code-signing-files/#generate-a-code-signing-certificate-using-xcode) to get one. -->
-
-<!-- You can still work with the generated binary locally at `packages/debug-bun-*/bun-debug` even if the code signing fails. -->
+The binary will be located at `packages/debug-bun-{platform}-{arch}/bun-debug`. It is recommended to add this to your `$PATH`.
 
-### Testing
+If you're building on a macOS device, you'll need to have a valid Developer Certificate, or else the code signing step will fail. To check if you have one, open the `Keychain Access` app, go to the `login` profile and search for `Apple Development`. You should have at least one certificate with a name like `Apple Development: user@example.com (WDYABC123)`. If you don't have one, follow [this guide](https://ioscodesigning.com/generating-code-signing-files/#generate-a-code-signing-certificate-using-xcode) to get one. You can still work with the generated binary locally at `packages/debug-bun-*/bun-debug` even if the code signing fails.
```

Comment:
This is no longer true. They can just skip it and it should work

Reply:
oh, that's wy this was commented out before